### DISCUSSION TITLE
Is there a pgdriver.Option that allows us to specify sslmode=disable?

### DISCUSSION THREAD

**Discussion by jhhom** [uptrace/bun#734] [Q&A] [answered]
I'm looking for a `pgdriver.Option` to specify `sslmode=disable`, but couldn't find any in the docs

**Comment by vmihailenco** [comment, marked as answer]
Try https://pkg.go.dev/github.com/uptrace/bun/driver/pgdriver#WithInsecure

**Reply by jhhom** [reply]
works, thank you.
